crates/uv/src/commands/venv.rs

```diff
@@ -188,5 +189,22 @@ async fn venv_impl(
         }
     }
 
+    match &platform.os() {
+        Os::Windows => {
```

Comment:
what about cmd users? 
- https://github.com/astral-sh/uv/pull/1523

Reply:
I wonder if we can detect the current terminal type in Windows...

Reply:
(or if we should just show both?

Reply:
Not sure if we can detect the current terminal type in Windows correctly/in a reliable way, would showing both like this be enough?

```bash
Activate with:
- Powershell: .\.venv\Scripts\activate.ps1
- CMD: .\.venv\Scripts\activate.bat
```

Reply:
@AlexWaygood / @MichaReiser can you take over review of this?

Reply:
I believe typing `.\.venv\Scripts\activate` (omitting the file extension) _should_ actually work on both PowerShell and CMD

Reply:
Nice, that's much simpler